content/docs/command-reference/remote/modify.md

```diff
@@ -1202,15 +1202,21 @@ by HDFS. Read more about by expanding the WebHDFS section in
   1. `user` parameter set with this command (found in `.dvc/config`);
   2. User defined in the URL (e.g. `webdavs://user@example.com/endpoint/path`)
 
-- `password` - password for WebDAV server, can be empty in case of using `token`
-  authentication.
+- `custom_auth_header` - HTTP header field name to use for authentication. Value
+  is set via `password`.
+
+  ```cli
+  $ dvc remote modify --local myremote custom_auth_header 'My-Header'
+  ```
```

Comment:
What's the default value? Do we need/recommend `--local` for this? Thanks

Reply:
There's no default value. And, `--local` is not strictly required since there are no secret values. But we already have similar snippet above in `http`, so I think let's keep this.

Reply:
By default, it won't be set. I am not sure - I picked this up from the documentation of the http remote, which has the same option (custom_auth_header) 
![image](https://user-images.githubusercontent.com/70193035/216373535-0738846e-0345-4a0e-ae85-71bf771dc171.png)